docs/docs/launch-manual-v2/sections/authoring-packages/sections/publishing-a-package.md

```diff
@@ -0,0 +1,71 @@
+
+### Publishing Your Package
+
+Before you publish a package it is a good idea to check ahead of time if a
+package with the same name has already been published to
+[the Pulsar Package Repository](https://web.pulsar-edit.dev/packages). You can
+do that by visiting `https://web.pulsar-edit.dev/packages/your-package-name` to
+see if the package already exists. If it does, update your package's name to
+something that is available before proceeding.
+
+Now let's review what the `pulsar -p publish` command does:
+
+1. Registers the package name on Pulsar Package Repository if it is being
+   published for the first time.
+2. Updates the `version` field in the `package.json` file and commits it.
+3. Creates a new [Git tag](https://git-scm.com/book/en/Git-Basics-Tagging) for
+   the version being published.
+4. Pushes the tag and current branch up to GitHub.
+5. Updates Pulsar Package Repository with the new version being published.
+
+Now run the following commands to publish your package:
+
+```sh
+$ cd path-to-your-package
+$ pulsar -p publish minor
+```
+
+If this is the first package you are publishing, the `pulsar -p publish` command may prompt you for your GitHub username and password. If you have two-factor authentication enabled, use a [personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) in lieu of a password. This is required for PPM to automatically manage and push tags to your package's GitHub repo and is needed only during the first time publishing. The credentials are stored securely in your [keychain](<https://en.wikipedia.org/wiki/Keychain_(software)>) once you login.
+
+Additionally if this is your first time publishing to the Pulsar Package Registry, you also need to provide PPM an API token to access your Pulsar User Account. If you don't have a Pulsar User Account already, you'll need to make one on the [Pulsar Website](https://web.pulsar-edit.dev/login). You can learn more about creating an account on the [Pulsar Website Docs](). Once you have successfully created an account copy the API token available on your user page and use `pulsar -p --login <YOUR_PULSAR_API_TOKEN>` to give PPM your API Key. Again just like your GitHub credentials this API Key is stored securely in your [keychain](<https://en.wikipedia.org/wiki/Keychain_(software)>).
+
+Now your package is published and available on the Pulsar Package Registry. Head on over to `https://web.pulsar-edit.dev/packages/your-package-name` to see your package's page.
+
+With `pulsar -p publish`, you can bump the version and publish by using
+
+```sh
+$ pulsar -p publish <version-type>
+```
+
+Where `version-type` can be `major`, `minor`, and `patch`.
+
+- **MAJOR** version when you make incompatible API changes.
+- **MINOR** version when you add functionality in a backwards compatible manner.
+- **PATCH** version when you make backwards compatible bug fixes.
```

Comment:
reminds me of this one rust comment. maybe this formulation is simpler:

major - upgrading breaks
minor - downgrading breaks
patch - neither

Reply:
this formulation isn't exactly the same as semver but it does make things clearer I think